src/routes/login/login.svelte

```diff
@@ -0,0 +1,59 @@
+<script lang="ts">
+  export let username:string = '';
+  export let password:string = '';
```

Comment:
Although synatically correct, placing a space before the type is a widely used convention to increase readability.

Reply:
I would consider using a prettyfier. Such plugins do exist in VSCode.